packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -47,37 +46,29 @@ interface Props {
   isOutgoingPaymentRequest: boolean
 }
 
-interface StateProps {
-  recipientInfo: RecipientInfo
-  showSendToAddressWarning: boolean
-}
+function RecipientPicker(props: RecipientProps) {
+  const recipientInfo = useSelector(recipientInfoSelector)
```

Comment:
nit: If you import `useSelector` from `src/redux/useSelector` instead of `react-redux` you don't need to set the `RootState` type.